src/ng-chat/components/ng-chat-window/ng-chat-window.component.css

```diff
@@ -203,9 +203,15 @@
                     }
 
 /* Mobile viewport only */
-@media only screen and (max-width: 581px) { 
+@media only screen and (max-width: 581px) {
     .ng-chat-window
     {
         position: initial;
     }
 }
+
+
+.responsive-image {
```

Comment:
Should this be added inside `.ng-chat-window` @rrpuente ? Something like `.ng-chat-window .image-message`?

Reply:
yes, it is better like that, Im on it.